nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -175,9 +181,17 @@ protected override void Awake()
 #if UNITY_IOS
             _commandLineOptions = CommandLineOptions.Load(Platform.GetStreamingAssetsPath("clo.json"));
 #else
-            _commandLineOptions = CommandLineOptions.Load(
-                CommandLineOptionsJsonPath
-            );
+            if (useLocalHeadless || useLocalMarketService)
```

Comment:
```suggestion
            if (useLocalMarketService)
```
Or you can just check `useLocalMarketService` only.